### ISSUE TITLE
Form Block: Sender IP Address is missing from Response Data section

### ISSUE THREAD

**Issue by jorpdesigns** [Automattic/jetpack#29124]
### Quick summary

Looks like there has been a recent update to the Form block where the sender's IP address is no longer displayed among the response data on the Feedback page. 

I am not sure if this change was intentional or not but having not found any information that proves it was intentional, I thought to create a bug report. 

Note that the IP address is still shared via email but I believe users should still be able to access this information from the website. Also since spam submissions don't appear to be sent via email, there is no way to note the IP address and blacklist.

### Steps to reproduce

1. Open a test site. The plan doesn't seem to matter as I could reproduce on Personal & Business (Atomic).
2. Create a page and add a form with a couple of fields. Fields choice doesn't seem to matter.
3. Save the page, fill the form on the front end and submit.

### What you expected to happen

The IP address should appear among the response data on the Feedback page.

You can find pictorial confirmation of this from 7 days ago: p1676410079891579/1676409312.172899-slack-C03TY6J1A

![Screen Shot on 2023-02-21 at 16:45:15](https://user-images.githubusercontent.com/40006288/220392204-ade33824-4949-4e49-be87-df5b7b654112.png)

### What actually happened

The IP Address is missing from the response data.

![Screen Shot on 2023-02-21 at 16:44:33](https://user-images.githubusercontent.com/40006288/220392073-c8e25887-69d1-40bc-8e20-fe970cfb1165.png)

### Impact

Most (> 50%)

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Simple, Atomic

### Logs or notes

Reported in 40470159-hc. Follow up: 5978129-zd-woothemes

User normally blacklists the IP addresses for spam submissions in order to minimise spam. I have explained this is not required (as Akismet should do this already) and also suggested other options for moderating submissions but user is really committed to fighting spam manually.

**Comment by kavyagokul** [comment]
📌 SCRUBBING : RESULT
- Tested on Simple - Replicated 
- Tested on AT - Replicated 
- Tested on Self-hosted - Replicated 
- Replicable on Core - Yes

📌 ACTIONS
- Marked as Triaged for Quality Squad review

**Comment by wynwin** [comment]
41393970-hc

**Comment by bobmatyas** [comment]
Another report: https://wordpress.org/support/topic/sender-ip-in-comments

**Comment by rmhall** [comment]
This issue has been open for well over a year at this point, so I made a quick fix. No time or inclination to submit a formal pull request or determine if it aligns with any WordPress coding standards (just quick and dirty to get going), but here it is for anyone who wants to quick copy and paste a single line of code to a single file to restore the missing IP display row – or use this as a general reference point to investigate more deeply where things are going wrong/broke and implement properly.

Edit this file:
/wp-content/plugins/jetpack/jetpack_vendor/automattic/jetpack-forms/src/contact-form/class-admin.php (Reference: https://raw.githubusercontent.com/Automattic/jetpack/trunk/projects/packages/forms/src/contact-form/class-admin.php )

Insert the following line of code at line 740:

`$content_fields["_feedback_ip"] = substr($chunks[0], strrpos($chunks[0], "IP:")+3);`

Now the Feedback Form Responses admin page will now show the missing “IP” row with the IP address of the form submission properly again. Located just below the horizontal rule in the “Response Data” column, and directly before the “Source” row – like it used to be as illustrated in the image above in the initial issue submission.

Basically it looks like the $content_fields array is not getting the _feeedback_ip pushed in properly. Didn’t have time to go back and look at commit history to determine the cause/where it broke originally, and yes, its kludgy without error checking, but it works in a pinch, and might help someone to finish this off properly with a jumping off point. For reference, made a slightly longer post about this on the discussion forums here: https://wordpress.org/support/topic/sender-ip-in-comments/

FWIW - User Agent would be really handy to display here as well.

**Comment by jeherve** [comment]
@rmhall Thanks for the comment. That's indeed an issue that we haven't taken the time to address until today. I took a look at it and open a Pull Request with an approach somewhat similar to yours, we'll get this merged for the next release.

**Comment by rmhall** [comment]
Awesome - look forward to it - thanks!
